services/core-web/common/customAxios.js

```diff
@@ -26,7 +26,7 @@ const CustomAxios = ({ errorToastMessage, suppressErrorNotification = false } =
 
       const status = error.response ? error.response.status : null;
       if (status === UNAUTHORIZED || status === MAINTENANCE) {
-        window.location.reload(false);
+        // window.location.reload(false);
```

Comment:
```suggestion
        window.location.reload(false);
```

Reply:
I think we can restore this (likely removed while testing is my guess)

Reply:
restored.